### DISCUSSION TITLE
Implemention details about config parameters

### DISCUSSION THREAD

**Discussion by geon314159** [ANLAB-KAIST/KENSv3#247] [Project Q&A 3. Internet Layer (IP/Routing)]
According to #232, #233,  PA4 do not simulate link failures or state changes and Without implementing the Hello&TLU time interval multicast, we can pass the testcases.

And Our team pass all the testcases (+ many random seeds) without using helloint, lsuint, timer. In this case, can we get full score by just passing all test cases?

**Comment by Pusnow** [comment]
Try to implement as much as you can for your own education.
We wil give you full credit if you pass all test cases and submit readme.txt well.
